### PR TITLE
Add tooltip to network graph.

### PR DESCRIPTION
**Pull request by kurnevsky**
Tooltip with `defaultNetFormat` from `NetworkMonitor`.

### CODE REVIEW

File: src/System/Taffybar/Widget/NetworkGraph.hs [hunk 1]
```diff
@@ -20,4 +25,9 @@ networkGraphNew config interfaces = do
       toLogScale = logScale 2 (2 ** 32)
       toSample (up, down) = map (toLogScale . fromRational) [up, down]
       sampleBuilder = return . toSample . getUpDown
-  channelGraphNew config chan sampleBuilder
+  widget <- channelGraphNew config chan sampleBuilder
+  void $ channelWidgetNew widget chan $ \speedInfo ->
+    let (up, down) = sumSpeeds $ map snd speedInfo
+        tooltip = showInfo defaultNetFormat 3 (fromRational down, fromRational up)
```
Comment (IvanMalison) [inline src/System/Taffybar/Widget/NetworkGraph.hs:31]: shouldn't we let this format be configurable?

Comment (kurnevsky) [reply src/System/Taffybar/Widget/NetworkGraph.hs:31]: Yes, but for this we need to change the signature of `networkGraphNew`. Is it ok? Or probably create a new function.

Comment (IvanMalison) [reply src/System/Taffybar/Widget/NetworkGraph.hs:31]: okay this confused me for a very long time when reading this:

When you name something like this tooltip, it makes it seem like it is some type of widget. The name should make it clear that this is just a string

**Review by IvanMalison — APPROVED** [review]

File: src/System/Taffybar/Widget/NetworkGraph.hs (hunk 1 above)
Comment (IvanMalison) [reply src/System/Taffybar/Widget/NetworkGraph.hs:31]: > Yes, but for this we need to change the signature of `networkGraphNew`. Is it ok? Or probably create a new function.

Right just add a parameter and rename this function. Then make a new function with the same name as this one and provide the default configuration calling the function with the added parameter.

Comment (kurnevsky) [reply src/System/Taffybar/Widget/NetworkGraph.hs]: Added new function with config.

**Pull request merged**